packages/angular/projects/ui-angular/src/lib/primitives-poc/password-field/password-field-component/password-field.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="amplify-flex amplify-field amplify-textfield">
+  <!-- Label -->
+  <label [for]="inputId" *ngIf="!labelHidden">
+    <ng-content select="[amplify-label]"> </ng-content>
+  </label>
+
+  <!-- Description Text -->
+  <ng-content select="[amplify-description-text]"></ng-content>
+
+  <!-- Actual Input -->
+  <div
+    class="amplify-flex amplify-field-group amplify-field-group--horizontal"
+    data-orientation="horizontal"
+  >
+    <div
+      class="amplify-field-group__field-wrapper amplify-field-group__field-wrapper--horizontal"
+      data-orientation="horizontal"
+    >
+      <ng-content select="[amplify-password]"></ng-content>
```

Comment:
idea: what do you think about calling this `amplify-password-input`? I think `amplify-password` name is a little bit ambiguous (field? input?).

Reply:
Yes, make it as amplify-password-input make this more perfect.

Reply:
Done.